SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -483,5 +486,7 @@ internal static void PlatformUnsetInstance()
         {
             Instance = null;
         }
+
+        public string ErrorReportID { get; set; }
```

Comment:
The ErrorReport object that is exposed in crash callbacks already has an Id property.

A static property here would not expose handled error ID but only the first pending crash from last run so it duplicates an existing feature and is not solving the feature request which is about handled errors.

What needs to be done instead is a modification on trackError instead to return the ID instead of void.

Reply:
updated it now track trace returns a guid and andusted the method to return errorId or empty Guid